pkg/devfile/adapters/docker/component/utils.go

```diff
@@ -49,14 +49,24 @@ func (a Adapter) createComponent() (err error) {
 	// Loop over each component and start a container for it
 	for _, comp := range supportedComponents {
 		var dockerVolumeMounts []mount.Mount
-		for _, vol := range a.componentAliasToVolumes[comp.Container.Name] {
 
-			volMount := mount.Mount{
-				Type:   mount.TypeVolume,
-				Source: a.volumeNameToDockerVolName[vol.Name],
-				Target: vol.ContainerPath,
+		// Let's find the container / component alias!
+		for _, vol := range a.componentAliasToVolumes {
```

Comment:
Seems like this and https://github.com/openshift/odo/pull/3409/files#diff-ec2e16874aae4e6d1f0ff0deefac8e04R106 are duplicates, can extract this into function and DRY the code?